src/main/java/nl/tudelft/pixelperfect/event/EventParameter.java

```diff
@@ -0,0 +1,72 @@
+package nl.tudelft.pixelperfect.event;
+
+/**
+ * Class representing a single parameter of an Event, having either a discrete value
+ * (EventParameterValue) or a continuous one (Integer).
+ * 
+ * @author Jesse Tilro
+ *
+ */
+public class EventParameter {
+
+  private String key;
+  private EventParameterValues value;
+  private int number;
```

Comment:
Only remark I have is that you have these two paremters, number and value. But then in the constructor you assign this.number = value

`  public EventParameter(String key, int value) {
     this.key = key;
    this.value = EventParameterValues.GENERIC;
    this.number = value;
  }` while correct this looks confusing. I suggest replacing this by this.number = number